less/home-page.less

```diff
@@ -14,16 +16,40 @@
   
   }
 
+  .space {
+    @media @phone {
+      width: 100%;
+    }
+  }
+
   .space-images {
     display: flex;
     flex-wrap: wrap;
     width: 100%;
 
+    @media @phone {
+      flex-wrap: nowrap;
+      flex-direction: column;
+    }
```

Comment:
Good job resetting wrap default, but was it breaking without adding or could you skip changing the wrap on the mobile since you changed it to column?